decentralization.adoc

```diff
@@ -26,23 +25,23 @@ trusted authorities.
 
 === Miner decentralization
 
-There have been attempts at creating digital moneys before Bitcoin,
-but most of them failed due to lack of decentralization of governance
+There have been attempts at creating digital currencies before Bitcoin,
+but most of them failed due to a lack of decentralization of governance
 and censorship resistance.
 
 Miner decentralization in Bitcoin means that the _ordering of
 transactions_ isn't carried out by a single entity or fixed set of
 entities. It's carried out collectively by all participants who want
-to; this collective of miners is a dynamic set of users. Anyone can
-join or leave as they wish. This is what makes Bitcoin censorship-resistant.
+to; this miners' collective is a dynamic set of users. Anyone can
+join or leave as they wish. This property makes Bitcoin censorship-resistant.
 
 If Bitcoin were centralized, it would be vulnerable to those who wish
 to censor, such as governments. It'd meet the same fate as earlier
 attempts to create digital money. In
 https://www.blockstream.com/sidechains.pdf[a paper] titled "`Enabling
 Blockchain Innovations with Pegged Sidechains`", the authors explained
-in their introduction how early versions of digital money weren't
-equipped for an <<adversarialthinking,adversarial environment>>:
+in their introduction, how early versions of digital money weren't
+equipped for an <<adversarialthinking, adversarial environment>>:
```

Comment:
This is a cross-reference with a custom text. I prefer to not have the space in front of the custom text.